docs/changelog.rst

```diff
@@ -10,8 +10,12 @@ v1.1.0 (UNRELEASED)
 Core API
 --------
 
-- Calling :meth:`mopidy.core.library.LibraryController.search`` with ``kwargs``
-  as the query is no longer supported (PR: :issue:`1090`)
+- Calling the following methods with ``kwargs`` is no longer supported.
```

Comment:
Don't mix "deprecated" and "not supported", they might not mean the same to everyone. I read "not supported" as "no longer works".